anesthetic/plot.py

```diff
@@ -677,6 +680,7 @@ def scatter_plot_2d(ax, data_x, data_y, *args, **kwargs):
     xmax = kwargs.pop('xmax', None)
     ymin = kwargs.pop('ymin', None)
     ymax = kwargs.pop('ymax', None)
+    kwargs = cbook.normalize_kwargs(kwargs, mlines.Line2D)
```

Comment:
This is the way `matplotlib` handles aliases such as `ms` and `markersize`, so this fixes #79.

As long as we `pop` all needed anesthetic specific kwargs before applying `cbook.normalize_kwargs`, this shouldn't cause any conflicts.

Might be worth applying before all the major matplotlib calls such as `plt.plot`, `plt.contour`, `plt.hist` etc. What do you think, @williamjameshandley?

Reply:
This sounds like an excellent idea. You could either have a go at doing it as part of this PR, or raise it as a new issue. I will approve and leave it up to you whether to merge or continue with `normalise_kwargs`

Reply:
> Might be worth applying before all the major matplotlib calls such as `plt.plot`, `plt.contour`, `plt.hist` etc.

Actually, `markersize` was the only one so far, where this applies.

Reply:
Also, `normalize_kwargs` is not as straight forward for contourplots or histograms. 

For `plt.plot` you need to pass `matplotlib.lines.Line2D` as a dictionary for all possible kwargs. This case was easy, as this is done at the start of [`plt.plot`](https://matplotlib.org/3.2.1/_modules/matplotlib/axes/_axes.html#Axes.plot) itself.

I couldn't find what we'd need to use in place of `Line2D` to make this work for `plt.contour`.